content/swift/concepts/dictionaries/terms/contains/contains.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.contains'
+Description: 'a Boolean value that indicates whether a dictionary contains a specified key-value pair.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Dictionaries'

```

Comment:
```suggestion
  - 'Dictionary'

```
`Dictionaries` is not present in the [tags.md file, ](documentation/tags.md)whereas `Dictionary` is present so you can use this as a more relevant one.